internal/expr/parser_helper_test.go

```diff
@@ -132,3 +132,47 @@ func (s *ExprInternalSuite) TestUnfinishedQuote(c *C) {
 		}
 	}
 }
+
+func (s *ExprInternalSuite) TestValidComments(c *C) {
+	var p = NewParser()
+
+	validComments := []string{
+		`-- Simple comment`,
```

Comment:
Maybe call it "Single-line"comment, which may be more descriptive :)

Reply:
Changed.